yarn.lock

```diff
@@ -19176,6 +19225,21 @@ video-extensions@^1.2.0:
   resolved "https://registry.yarnpkg.com/video-extensions/-/video-extensions-1.2.0.tgz#62f449f403b853f02da40964cbf34143f7d96731"
   integrity sha512-TriMl18BHEsh2KuuSA065tbu4SNAC9fge7k8uKoTTofTq89+Xsg4K1BGbmSVETwUZhqSjd9KwRCNwXAW/buXMg==
 
+viem@^1.0.0:
```

Comment:
I'm a little worried this will behave differently, because it is no longer working with ethers. Have you tested a connected gnosis safe?

Reply:
oh wow i did not notice that

Reply:
have just tested it, and it worked seamlessly: [https://app.safe.global/transactions/tx?id=multisig_0xb1bE9cC6cC7159fEBfEaB69D9a94fa[…]b0dcec69f&safe=gor:0xb1bE9cC6cC7159fEBfEaB69D9a94fae03D7828C3](https://app.safe.global/transactions/tx?id=multisig_0xb1bE9cC6cC7159fEBfEaB69D9a94fae03D7828C3_0x7e0d1897ce204edf4d2ff0166d96fd192989ce8eea46c57dac4d615b0dcec69f&safe=gor:0xb1bE9cC6cC7159fEBfEaB69D9a94fae03D7828C3)

Reply:
We only used ethers to generate an EIP-191 message/hashing eip712 data, which is covered by SDK tests. It shouldn't require any changes/considerations on the dapp side when upgrading. All the APIs stayed the same

Reply:
Also, just FYI, we spun out of gnosis, so we're just safe now. Ideally we need to coordinate the package name upgrade too :wink: